### ISSUE TITLE
lapis.tube times out

### ISSUE THREAD

**Issue by Mennaruuk** [blubbll/lapis-tube#11]
https://isitup.org says site is up:
![image](https://user-images.githubusercontent.com/52135169/106399376-2fc50100-63e6-11eb-9d22-4af1e170ea2b.png)

However, this is what I get on Firefox:
![image](https://user-images.githubusercontent.com/52135169/106399383-3fdce080-63e6-11eb-939b-f9836a42de11.png)

**Comment by Mennaruuk** [comment]
Now it spits out a 404:
![58041885-46FB-469B-83BD-9B359542B4DD](https://user-images.githubusercontent.com/52135169/106490319-b24fce00-6483-11eb-8fbc-35bcb8f3127b.png)

**Comment by blubbll** [comment]
Yeah i just renewed the domain, it was expired. I currently don't have too much time to maintain this project. But i have plans on re-using the concepts once more and making this more stable. Since the original invidiou.us instance is now down and youtube sometimes marking myself as a bot im not sure if theres a future rrgarding that part of the api though. But seeing yt kinda works behind tor, i might make an own backend service when i continue this project. Also the translation service subscription expired, i would also need to continue there if you want different ui languages. However, i renewed the domain now so it will be usable for another year.

**Comment by Mennaruuk** [comment]
Thank you!!! Whatever decision you make is completely respected. I know there are front ends like [ViewTube](https://dev.viewtube.io) and [CloudTube](https://tube.cadence.moe) that permit instance selection within the settings so users can choose which instance to watch from if one goes down. I think that’d be one good way to bypass this issue.

I’m surprised, however, that you needed a translation subscription. I know a service like [AllTube](https://alltubedownload.net/) has a little flag where you can select the preferred language. Maybe the translations were crowdsourced. If this project happens to come back up, do you think crowdsourcing translations would be a better alternative than subscribing to this UI translation company?

Best of luck!!!!!
